src/strategies/dvision-dao/index.ts

```diff
@@ -0,0 +1,22 @@
+import fetch from 'cross-fetch';
+
+export const author = 'dvision';
+export const version = '0.0.1';
+
+export async function strategy(space, network, provider, addresses, options) {
+  const score = {};
+  const response = await fetch(
+    `${options.url}/api/votepower?wallet=${addresses[0]}&network=${network}`,
```

Comment:
Hi @Sotatek-VietNguyen I think you can use `api` strategy here , did you give it a try? https://snapshot.org/#/strategy/api